src/java/picard/fingerprint/CheckFingerprint.java

```diff
@@ -89,6 +89,10 @@
 
     private final Log log = Log.getInstance(CheckFingerprint.class);
 
+    public static final String FINGERPRINT_SUMMARY_FILE_SUFFIX = "fingerprinting_summary_metrics";
```

Comment:
@eitanbanks this wont work long term, since some environments (like BaseSpace) do not allow long file name suffixes.  Yes, that's bad, and we could say tough luck, fix your environment, but unfortunately, that's tough to do (especially for BaseSpace).    If you look at other tools who produce summary and detail metrics (ex `CollectGcBiasMetrics`) we pass in two command line options.